subsys/usb/class/bluetooth.c

```diff
@@ -120,34 +88,31 @@ static void hci_tx_thread(void)
 	}
 }
 
-static void bluetooth_bulk_out_callback(u8_t ep,
-					enum usb_dc_ep_cb_status_code ep_status)
+static void acl_read_cb(u8_t ep, int size, void *priv)
 {
-	struct net_buf *buf;
-	u32_t len;
+	struct net_buf *buf = priv;
 
-	SYS_LOG_DBG("ep %x status %d", ep, ep_status);
-
-	/* Read number of bytes to read */
-	usb_read(ep, NULL, 0, &len);
+	if (size > 0) {
+		buf->len += size;
+		bt_buf_set_type(buf, BT_BUF_ACL_OUT);
+		net_buf_put(&tx_queue, buf);
+		buf = NULL;
+	}
 
-	if (!len || len > BT_BUF_ACL_SIZE) {
-		SYS_LOG_ERR("Incorrect length: %u\n", len);
-		return;
+	if (buf) {
+		net_buf_unref(buf);
 	}
 
 	buf = net_buf_alloc(&acl_tx_pool, K_NO_WAIT);
 	if (!buf) {
```

Comment:
Is not it issued each time it is called from USB_DC_CONFIGURED?  `if (!buf && size)` ?